CREDITS.md

```diff
@@ -0,0 +1,52 @@
+Credits
+=======
+
+This project makes use of Open Source components. Below is a list of these components included in this project's source code, and their license information. This project also uses js packages released by NPM, see [package.json](/package.json). Source code and license information for each of these packages is available at https://npmjs.org. Many thanks to all of the original authors!
+
+### https://github.com/facebook/react
```

Comment:
We shouldn’t be including either of these libraries in the built files, so I don’t think we need this

Reply:
Oh? Is that because we're relying on Gutenberg to bring them over?

Reply:
Removed credits for now, until we actually need them, see https://github.com/woocommerce/wc-admin/pull/748/commits/ecc43a60ec78db2cf0d87c003be7a6674408f3dc

Reply:
Yeah, anything react should be provided by `@wordpress/element` so we're not loading multiple Reacts. It looks like we're not using `react-redux` at all, either (just `redux`).